com.unity.render-pipelines.high-definition/Documentation~/snippets/shader-properties/advanced-options/allow-radial-smoothness.md

```diff
@@ -0,0 +1,4 @@
+<tr>
+<td>**Allow Radial Smoothness**</td>
+<td>Adds a secondary **Radial Smoothness** block to the target, controlling the internal scattering of the light paths and absorption that occurs within the fiber. Enabling this option comes at a slight performance cost to the shader; when disabled, it will fall back to an approximation of the internal scattering.</td>
```

Comment:
<td>Adds a secondary **Radial Smoothness** block to the target that controls the internal scattering of light paths and the amount of light the hair fiber absorbs. Enabling this option makes the shader use slightly more resources on the GPU. When this property is disabled, the shader approximates internal scattering.</td>